.github/workflows/release.yml

```diff
@@ -22,37 +22,75 @@ jobs:
           token: ${{secrets.GITHUB_TOKEN}}
           default-branch: main
           release-type: simple
+    outputs:
+      release_created: ${{ steps.release.outputs.releases_created }}
+      release_tag_name: ${{ steps.release.outputs.release_tag_name }}
+      paths_released: ${{ steps.release.outputs.paths_released }}
```

Comment:
There's a bug in some versions of release please where this is a string, not a bool, so I suggest doing `fromJSON` for all of these, as in Java: https://github.com/open-feature/java-sdk-contrib/blob/f22e9e42f7739d9b59aa97ac37a080d901094e0d/.github/workflows/release-please.yml#L21